src/test/java/net/openhft/chronicle/queue/TestDeleteQueueFile.java

```diff
@@ -416,6 +416,17 @@ public void run() {
             Jvm.startup().on(TestDeleteQueueFile.class, "Tailer thread terminated: " + direction);
         }
 
+        private DocumentContext readingDocumentWithRetries(ExcerptTailer excerptTailer) {
+            DocumentContext documentContext = null;
+            for (int i = 0; i < 2; i++) {
```

Comment:
Is there any significance to the number chosen `2` here?

Reply:
The first call to `readingDocument()` will return a not-present document context when a deleted cycle is encountered, but it will invalidate the cached cycle tree, so a subsequent call will rebuild that from disk and be able to skip forward to the next actually-present cycle. There's probably a small chance that it hits a deleted cycle twice in a row, but maybe we monitor that and see how likely it is.